.gitignore

```diff
@@ -1,4 +1,20 @@
-# Compiled class file
+# Ignore bin folders at the root of each project
+/*/bin/
+/*/bin_test/
+
+# Ignore build folders at the root of each project
+/*/build/
+
+# Ignore generated folders at the root of each project
+/*/generated/
+
+# Ignore gradle files and folders at the root of each project
+/*/.gradle/
+/*/gradle/
+/*/gradlew
+/*/gradlew.bat
```

Comment:
Since I expect this repository will hold more than one project at some point I would suggest checking in the gradle wrapper at the root of the project so that each project doesn't need their own. Something like: 
```
/ -root
gradlew
gradlew.bat
build.gradle - main build file
settings.gradle - lists subprojects
io.openliberty.java.internal_fat_misc/
     build.gradle --build guidelines for subproject
```